Sources/Purchasing/Offerings.swift

```diff
@@ -93,4 +110,50 @@ public extension Offerings {
         return description
     }
 
+    /**
+     Retrieves a current offering for a placement identifier, use this to access offerings defined by targeting
+     placements configured in the RevenueCat dashboard, 
+     e.g. `offerings.getCurrentOffering(forPlacement: "placement_id")`.
+     */
+    @objc(getCurrentOfferingForPlacement:)
+    func getCurrentOffering(forPlacement placementIdentifier: String) -> Offering? {
```

Comment:
is it more swift-style to make this `(for placement placementIdentifier: String)` instead? 
so the callsite is 
```swift 
getCurrentOffering(for: "HomePageBanner")
```

instead of

```swift 
getCurrentOffering(forPlacement: "HomePageBanner")
```

or even 

```swift
currentOffering(for: "HomePageBanner")
```

Reply:
we've historically reserved verbs like `get` or `fetch` as ways to indicate that it'll issue some work like network requests